front-end/src/main.ts

```diff
@@ -0,0 +1,12 @@
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
```

Comment:
The same

Reply:
Because it separates external libs from internals one.